PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -743,15 +743,15 @@ LRESULT CALLBACK ScintillaEditView::ScintillaProc(
 							}
 						}
 
-						pScint->execute(SCI_BEGINUNDOACTION);
-
-						// Let Scitilla do its job, if any
+						// Let Scintilla do its job, if any
 						if (nbCaseForScint > 0)
 							break;

```

Comment:
replacing with `::DefSubclassProc(hWnd, uMsg, wParam, lParam);` should be enough instead of "moving"  SCI_BEGINUNDOACTION and SCI_ENDUNDOACTION

Reply:
@ozone10 
Indeed. That solves the problem. However, it is always better to keep SCI_BEGINUNDOACTION and SCI_ENDUNDOACTION as closer as passible.

@xomx 
Could you keep the latest change (solution of @ozone10) and restore the 2 UNDOACTION into the `if (!isColumnSelection)` condtion please?
I know you're planning to implement scoped automatic UNDOACTION (great idea BTW), in the meanwhile, let's reduce the distance of 2 UNDOACTION, to avoid next pontential regression.